lib/src/modules/conditional.dart

```diff
@@ -617,15 +616,12 @@ class Case extends Conditional {
   String get caseType => 'case';
 
   @override
-  Set<Logic> execute() {
-    final drivenLogics = <Logic>{};
-
+  void execute(Set<Logic> drivenSignals) {
     if (!expression.value.isValid) {
       // if expression has X or Z, then propogate X's!
       for (final receiver in getReceivers()) {
         receiverOutput(receiver).put(LogicValue.x);
       }
-      return {};
```

Comment:
You still want to return here, but just nothing (`return;`) because you dont want to continue down